{
  "containers": {
    "adp": [
      {
        "providerMetadata": {
          "orgId": "af854a3a-2127-422b-91ae-364da2661108",
          "shortName": "CVE",
          "dateUpdated": "2024-08-03T04:36:06.656Z"
        },
        "title": "CVE Program Container",
        "references": [
          {
            "tags": [
              "x_refsource_MISC",
              "x_transferred"
            ],
            "url": "https://security.snyk.io/vuln/SNYK-PYTHON-OPCUA-2988730"
          },
          {
            "tags": [
              "x_refsource_MISC",
              "x_transferred"
            ],
            "url": "https://security.snyk.io/vuln/SNYK-PYTHON-ASYNCUA-2988731"
          },
          {
            "tags": [
              "x_refsource_MISC",
              "x_transferred"
            ],
            "url": "https://github.com/FreeOpcUa/python-opcua/issues/1466"
          }
        ]
      }
    ]
  },
  "cveMetadata": {
    "assignerOrgId": "bae035ff-b466-4ff4-94d0-fc9efd9e1730",
    "assignerShortName": "snyk",
    "cveId": "CVE-2022-25304",
    "datePublished": "2022-08-23T05:05:11.904697Z",
    "dateReserved": "2022-02-24T00:00:00",
    "dateUpdated": "2024-09-17T02:48:11.272Z",
    "state": "PUBLISHED"
  },
  "search": "解析验收测试文案"
}